lighthouse-cli/test/smokehouse/pwa-expectations.js

```diff
@@ -0,0 +1,128 @@
+    url: 'https://airhorner.com/',
+    audits: {
+      'is-on-https': true,
+      'redirects-http': true,
+      'service-worker': true,
+      'works-offline': true,
+      'manifest-display': true,
+      'manifest-exists': true,
+      'manifest-background-color': true,
+      'manifest-theme-color': true,
+      'manifest-icons-min-192': true,
+      'manifest-icons-min-144': true,
+      'manifest-name': true,
+      'manifest-short-name': true,
+      'manifest-start-url': true,
+      // 'cache-start-url': true
+    }
+  },
+
+  {
+    initialUrl: 'https://www.chromestatus.com/',
+    url: 'https://www.chromestatus.com/features',
+    audits: {
+      'is-on-https': true,
+      'redirects-http': true,
+      'service-worker': true,
+      'works-offline': false,
+      'manifest-display': true,
+      'manifest-exists': true,
+      'manifest-background-color': true,
+      'manifest-theme-color': true,
+      'manifest-icons-min-192': true,
+      'manifest-icons-min-144': true,
+      'manifest-name': true,
+      'manifest-short-name': true,
+      'manifest-start-url': true,
+      // 'cache-start-url': true
+    }
+  },
+
+  {
+    initialUrl: 'https://jakearchibald.github.io/svgomg/',
+    url: 'https://jakearchibald.github.io/svgomg/',
+    audits: {
+      'is-on-https': true,
+      'redirects-http': true,
+      'service-worker': true,
+      'works-offline': true,
+      'manifest-display': true,
+      'manifest-exists': true,
+      'manifest-background-color': true,
+      'manifest-theme-color': true,
+      'manifest-icons-min-192': true,
+      'manifest-icons-min-144': true,
+      'manifest-name': true,
+      'manifest-short-name': true,
+      'manifest-start-url': true,
+      // 'cache-start-url': true
+    }
+  },
+
+  {
+    initialUrl: 'https://shop.polymer-project.org/',
+    url: 'https://shop.polymer-project.org/',
+    audits: {
+      'is-on-https': true,
+      'redirects-http': true,
+      'service-worker': true,
+      'works-offline': true,
+      'manifest-display': true,
+      'manifest-exists': true,
+      'manifest-background-color': true,
+      'manifest-theme-color': true,
+      'manifest-icons-min-192': true,
+      'manifest-icons-min-144': true,
+      'manifest-name': true,
+      'manifest-short-name': true,
+      'manifest-start-url': true,
+      // 'cache-start-url': true
+    }
+  },
+
+  {
+    initialUrl: 'https://pwa.rocks',
+    url: 'https://pwa.rocks/',
+    audits: {
+      'is-on-https': true,
+      'redirects-http': true,
+      'service-worker': true,
+      'works-offline': true,
+      'manifest-display': true,
+      'manifest-exists': true,
+      'manifest-background-color': true,
+      'manifest-theme-color': true,
+      'manifest-icons-min-192': true,
+      'manifest-icons-min-144': true,
+      'manifest-name': true,
+      'manifest-short-name': true,
+      'manifest-start-url': true,
+      // 'cache-start-url': true
+    }
+  }
+];
```

Comment:
Some fairly static PWAs are listed here where we know the sources may be unlikely to majorly change (with the exception of ChromeStatus). Was this intentional? It might be useful to chat about separately smokehouse testing more complex/live apps, but I could see that being most useful before we cut a release.